src/main/java/com/coherentsolutions/pot/insuranceservice/controller/InsurancePackageManagementController.java

```diff
@@ -46,4 +47,9 @@ public InsurancePackageDto createInsurancePackage(
     return insurancePackageManagementService.createInsurancePackage(companyId, insurancePackageDto);
   }
 
+  @DeleteMapping("{id}")
```

Comment:
Minor: I see that for other similar endpoints a slash is used before the ID parameter. Let's be consistent here and keep the style.
```suggestion
  @DeleteMapping("/{id}")
```

Reply:
Ok, will change that in the next PR